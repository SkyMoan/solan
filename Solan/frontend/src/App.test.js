import React from 'react'
import renderer from 'react-test-renderer'

import ProfileEdit from './components/userProfile/user-profile-edit'
import ContactForm from './components/contactForm/contactForm'
import UserProfile from './components/userProfile/user-profile'
import Startups from './components/startups/startups'

it('render contact form correctly', () => {
  const tree = renderer.create(<ContactForm/>).toJSON();
  expect(tree).toMatchSnapshot();
})

it('render user profile correctly', () => {
  const tree = renderer.create(<UserProfile/>).toJSON();
  expect(tree).toMatchSnapshot();
})

it('render edit profile correctly', () => {
  const tree = renderer.create(<ProfileEdit/>).toJSON();
  expect(tree).toMatchSnapshot();
})

it('render startups correctly', () => {
  const tree = renderer.create(<Startups/>).toJSON();
  expect(tree).toMatchSnapshot();
})

