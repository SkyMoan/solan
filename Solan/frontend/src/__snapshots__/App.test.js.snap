// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render contact form correctly 1`] = `
<div
  className="main-form"
>
  <div
    className="content-div"
  >
    <div
      className="contact-us-description"
    >
      <p
        className="heading"
      >
         Ta kontakt 
      </p>
      <p>
         Send ditt spørsmål, så skal vi prøve å svare innen 1-2 dager 
      </p>
    </div>
    <div
      className="contact-details"
    >
      <p
        className="justify-content-center margin-top"
      >
         NAVN 
      </p>
      <input
        className="text-fields"
        onChange={[Function]}
        placeholder="Navn"
        type="text"
        value=""
      />
      <p
        className="justify-content-center margin-top"
      >
         EPOST 
      </p>
      <input
        className="text-fields"
        onChange={[Function]}
        placeholder="ole@domene.com"
        style={
          Object {
            "borderBottom": false,
          }
        }
        type="text"
        value=""
      />
      <p
        className="justify-content-center margin-top"
      >
         MELDING 
      </p>
      <textarea
        className="text-fields"
        onChange={[Function]}
        placeholder="Din spørsmål"
        value=""
      />
      <button
        className="submit-button"
        onClick={[Function]}
      >
        SEND MELDING
      </button>
    </div>
  </div>
</div>
`;

exports[`render edit profile correctly 1`] = `
<div
  className="edit-profile"
>
  <form
    onSubmit={[Function]}
  >
    <input
      name="email"
      onChange={[Function]}
      placeholder="E-post"
      type="email"
      value=""
    />
    <input
      name="firstName"
      onChange={[Function]}
      placeholder="Fornavn"
      type="text"
      value=""
    />
    <input
      name="lastname"
      onChange={[Function]}
      placeholder="Etternavn"
      type="text"
      value=""
    />
    <input
      name="bio"
      onChange={[Function]}
      placeholder="Bio"
      size="63"
      type="text"
      value=""
    />
    <input
      onClick={[Function]}
      type="submit"
    />
  </form>
</div>
`;

exports[`render startups correctly 1`] = `
<div
  className="startups-grid-container"
>
  <div
    className="startups_boxspacer"
  />
  <div
    className="startups_A"
  >
    <p
      className="startups_banner_image_text"
    >
      Startups
    </p>
  </div>
</div>
`;

exports[`render user profile correctly 1`] = `
<div
  className="profile-grid-container"
>
  <div
    className="profile_boxspacer"
  >
    <div
      className="profile-image"
    >
      <img
        alt="Profile Pic"
        style={
          Object {
            "width": "10%",
          }
        }
      />
    </div>
    <div
      className="profile-name"
    >
      Navn: 
       
    </div>
    <div
      className="profile-email"
    >
      Epost: 
    </div>
    <div
      className="profile-bio"
    >
      Bio: 
    </div>
    <div>
      <a
        href="/profile/edit"
        onClick={[Function]}
      >
        Rediger profil
      </a>
    </div>
  </div>
</div>
`;
